src/Components/Parameters/DefaultParameter.vue

```diff
@@ -0,0 +1,61 @@
+<!--
+  - @copyright Copyright (c) 2019 Joas Schilling <coding@schilljs.com>
+  -
+  - @author Joas Schilling <coding@schilljs.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
```

Comment:
For later we could also have those in a library, since they would be reusable then for any ui that uses the [rich object string definitions](https://github.com/nextcloud/server/blob/b5e9f7e846c8389d36c789d6b3a7c7a3c02e924d/lib/public/RichObjectStrings/Definitions.php) from the server

Reply:
yeah, specially the empty default would be nice I guess.